.devcontainer/devcontainer.json

```diff
@@ -11,7 +11,7 @@
     "ghcr.io/devcontainers/features/node:1": {
       "version": "18"
     },
-    "ghcr.io/devcontainers/features/docker-in-docker:1": {
+    "ghcr.io/devcontainers/features/docker-in-docker:2.9.0": {
```

Comment:
We should just use `2` as the feature version to get updates and fixes, and the "version": 20 below can be removed as the docker buildx issues has been fixed now